lib/blobs.js

```diff
@@ -14,17 +15,46 @@ import { assertOkResponse } from './http-assertions.js'
  */
 
 /**
- * Returns a list of recent confirmed blobs.
+ * @typedef {{
+ * content: Blob[],
+ * totalPages: number
+ * }} WalrusScanResponse
+ */
+
+/**
+ * Returns a list of certified blob Base64 IDs.
  *
- * @param {typeof globalThis.fetch} fetch The fetch function to use.
+ * @param {typeof globalThis.fetch} [fetch=globalThis.fetch] The fetch function to use.
  * @returns {Promise<string[]>} A list of recent blob Base64 IDs.
  */
-export const getRecentBlobs = async (fetch = globalThis.fetch) => {
-  const res = await fetch('https://walruscan.com/api/walscan-backend/testnet/api/blobs?page=0&sortBy=TIMESTAMP&orderBy=DESC&searchStr=&size=20')
-  await assertOkResponse(res, 'Failed to retrieve recent blobs')
-  const blobs = await res.json()
+export const getBlobs = async (fetch = globalThis.fetch) => {
+  // TODO: Filter out blobs by start and end epoch
+  const recentBlobsResponse = await fetchBlobs(0, fetch)
+  const recentBlobs = getCertifiedBlobIds(recentBlobsResponse.content)
+  const randomPage = pickRandomNumber(1, recentBlobsResponse.totalPages)
```

Comment:
how do you know these blobs aren't expired?

Reply:
I've managed to resolve it in https://github.com/CheckerNetwork/walrus-checker/pull/15/commits/b18035f4f9f92a4ea122bdee2515e87b4c897c93